services/webnn/ort/context_impl_ort.cc

```diff
@@ -185,7 +211,7 @@ SessionOptions::Create(const mojom::CreateContextOptions::Device device_type) {
     // 2 (extended optimization).
     // https://github.com/microsoft/onnxruntime/blob/89f8206ba4f1c22c39e0297fb55272e8ce8cd7d0/onnxruntime/core/session/inference_session.cc#L2003
     CALL_ORT_FUNC(ort_api->SetSessionGraphOptimizationLevel(
-        session_options.get(), GraphOptimizationLevel::ORT_ENABLE_ALL));
+        session_options.get(), kGraphOptimizationLevelParam.Get()));
```

Comment:
The override can be consolidated, duplicating for all EPs seems unnecessary.

Reply:
Updated, PTAL, thanks! @huningxin